src/renderer.rs

```diff
@@ -320,4 +325,71 @@ impl Renderer {
             line(color::BLACK.into(), 1., [start[0], start[1], end[0], end[1]], c.transform, g);
         }
     }
+
+    /// Save the drawings to SVG
+    fn save_svg<P: AsRef<path::Path>>(&mut self, path: P) {
```

Comment:
Given that we're always going to call this with one type, it's probably fine to change this function to take `path: &path::Path` instead of using generics.